src/App.js

```diff
@@ -0,0 +1,22 @@
+import './App.css';
+import './components/categories.css';
+import { Route, Routes } from 'react-router-dom';
+import BookList from './components/booklist';
+import Categories from './components/categories';
+import Navigation from './components/navigation';
+
+function App() {
+  return (
+    <>
+      <section className="bookstoresection">
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<BookList />} />
```

Comment:
- [ ] You are required to have a  `<Books>` component, which contains the following components: `<BookList>` which renders a list of books using the `<Book>` component. Following this structure, it is the `Books` component that should be imported into the highlighted piece of code. 👍 



<Book>

Takes the book data as a prop and renders the title and the author
Has a button for removing books